data/856/824/99/85682499.geojson

```diff
@@ -10,6 +10,12 @@
     "geom:latitude":52.501527,
     "geom:longitude":13.401859,
     "iso:country":"DE",
+    "label:deu_x_preferred_longname":[
+        "Land Berlin"
```

Comment:
Like we did in France and some other countries, let's strip the `Land ` prefix in the default longname label as it's heavy handed for native German speakers. So just `Berlin`. For all region labels.

Reply:
I'll take care of these in an upcoming commit. In some region cases, though, we have other placetypes, like "Freistaat", or in the case of Hamburg, "Freie und Hansestadt Hamburg". Would you expect those to be removed as well?